packages/@ngtools/benchmark/src/index.ts

```diff
@@ -0,0 +1,96 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as yargs from 'yargs';
+
+import { BenchmarkOptions } from './benchmark-options';
+import { benchmarkLog } from './utils';
+import { benchmark } from './benchmark';
+
+
+// Set options via yargs
+const benchmarkOptions: BenchmarkOptions = yargs
+  .usage('$0 [args]')
+  .options({
+    'command': {
+      description: 'Command to benchmark, defaults to benchmarking `ng serve` if not set',
+      type: 'string',
+      alias: 'c'
+    },
+    'iterations': {
+      description: 'Number of iterations to run benchmark',
+      type: 'number',
+      default: 5,
+      alias: 'i'
+    },
+    'extra-args': {
+      description: 'Extra arguments to combine and run',
+      type: 'array',
+      default: [],
+      alias: 'ea'
+    },
+    'match': {
```

Comment:
It varies a bit... on some tests I wanted to benchmark the difference in rebuilds between the main chunk and lazy loaded chunks. Or to check the rebuild time for css/html. In those cases the written string also changes.